storage/aptosdb/src/lib.rs

```diff
@@ -160,6 +169,48 @@ fn update_rocksdb_properties(db: &DB) -> Result<()> {
     Ok(())
 }
 
+#[derive(Debug)]
+struct RocksdbPropertyReporter {
```

Comment:
Its interesting that we didn't have any tests failing after this was accidentally removed. Seems like we don't have any relevant test for it? May be let's file a follow up issue for adding some tests?

Reply:
There's no interface coded for other threads to talk with it. Not sure it's worth adding one just for testing.. This affects only the dashboard anyway.

Reply:
Adding the issue is slower than adding the test :trollface: